applications/webhooks/src/webhooks_util.erl

```diff
@@ -628,22 +727,22 @@ available_events() ->
 
 -spec fetch_available_events() -> ne_binaries().
 fetch_available_events() ->
-    {'ok', MasterAccountDb} = kapps_util:get_master_account_db(),
-    View = <<"webhooks/webhook_meta_listing">>,
-    case kz_datamgr:get_all_results(MasterAccountDb, View) of
+    MasterAccountDb = get_master_account_db(),
```

Comment:
I don't see the advantage of this change. If the master account can't be fetched why should this work proceed?

Reply:
It won't proceed, `kz_datamgr:get_results/2` would return `{error,invalid_db_name}` because master account db is `undefined`

Reply:
But it will return `[]` instead of crashing, right?